protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -46,6 +46,13 @@ type Registry struct {
 
 	// mergeFileName target swagger file name after merge
 	mergeFileName string
+
+	// allowRepeatedFieldsInBody permits repeated field in body field path of `google.api.http` annotation option
+	allowRepeatedFieldsInBody bool
+
+	// useJSONNameInSwaggerDef if it true Field.GetJsonName() will be used for generating swagger definitions,
+	// otherwise Field.GetName() will be used
+	useJSONNameInSwaggerDef bool
```

Comment:
Please rename this into `useJSONNamesForFields` and update the comment appropriately :-)

Separately, can you then make `protoc-gen-grpc-gateway` use the JSON names as well if the flag is set? If not, can you please add a TODO here for someone to take a stab at it. It would be long-term important that `-swagger` defines the same API that `-grpc-gateway` exposes.